ci/tests/vmachine/vmachine_test.py

```diff
@@ -29,6 +30,7 @@
 VPOOL_NAME = test_config.get('vpool', 'vpool_name')
 TEMPLATE_SERVERS = ['http://sso-qpackages-loch.cloudfounders.com/templates/openvstorage', 'http://172.20.3.8/templates/openvstorage']
 
+DEVSTACK_INSTALLED = mgmtgeneric.is_devstack_installed()
```

Comment:
When running the test once without having DEVSTACK installed, the test will be skipped
When installing devstack and running the test again, the test will still be skipped ==> wanted behavior?